dist/web3.js

```diff
@@ -1444,6 +1444,9 @@ web3.setProvider = function (provider) {
     this.currentProvider = provider;
     RequestManager.getInstance().setProvider(provider);
 };
+web3.isConnected = function(){
```

Comment:
I want that dapps running in mist are still compatible with normal browsers, and there is no reason they shouldn't. So when you use the in a normal browser, the they have to inform the user to start a node. 

And if they aren't able to check the connection to a node, they will have be able to deal with unexpected response behaviour etc. So what will happen is that people will do things like checking for `eth.accounts` and then show a window, but there is no reason why we should not offer a standard to check for connections.